docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -1,120 +1,102 @@
-  
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs by **July 30, 2024**. If users don’t complete this by the deadline, they’ll be automatically removed from the bcgov GitHub organization including all the code repositories connected to [bcgov](#bcgov).   
+## Single Sign-On 
 
-Contractors without IDIRs must order one before SSO can be enabled. 
+To keep access to repositories in the `bcgov` enterprise, all users must be authenticated. Contractors without IDIRs must order one before SSO can be enabled. 
 
 [Learn more about IDIRs](https://dev.developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/). 
- 
-[Learn more about the timelines and instructions](bc-government-organizations-in-github.md#the-changes). 
   
-### The changes 
- 
-Before this upgrade, users only needed their personal GitHub IDs to access the bcgov GitHub organization as a member. 
  
-By **July 30, 2024**, users must follow prompts in GitHub to link their personal GitHub ID with their IDIR account.  If not, users will be automatically removed. 
+### Directions to link your account
 
-If the instructions to re-add users doesn't work, please submit a ticket at [Jira Service Manager](https://citz-do.atlassian.net/servicedesk/customer/portal/2)
- 
-[Learn more about how linking works](github-transition-guide.md#technical-information).  
+If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](github-transition-guide.md#idirs) before proceeding:
+
+1. [Authentiacate your IDIR](https://github.com/orgs/bcgov/sso)
+2. You'll get an email with an invitation to join the **bcgov** GitHub organization. The invitation will be sent to the email address associated with your GitHub account.
+3. Your teammates can add you to your team's repositories.
 
-### Costs
+If any technical errors happen, try to [re-authorize your personal access tokens and/or SSH keys](github-transition-guide.md#resetting-github-keys) so they continue to work with SSO enabled. 
  
-Ministries won’t be charged for upgraded GitHub licenses.  However, ordering IDIRs for contractors may cost and need expense approvals. We have more information about IDIRS on our [IDIR transition guide](github-transition-guide.md#idirs). 
+### SSO timelines for `bcgov-c` 
+
+
+We begin transition of [`bcgov-c`](#bcgov-c) into Enterprise GitHub on **August 7, 2024**. After migration completes, we'll enable SSO for [`bcgov-c`](#bcgov-c) on **August 21, 2024**, using the same process as the SSO enablement for `bcgov`.
   
-### Directions to link your accounts before July 30
+For current B.C. Government private organizations part of the legacy `bcgov-ent` Enterprise implementation, we’ll work with your respective Information Management Branches and Information Systems Branches to identify next steps.
```

Comment:
this content is not related to bcgov-c, so either needs another heading or the heading above needs to be more generic